streamer/src/quic.rs

```diff
@@ -114,9 +116,10 @@ pub(crate) fn configure_server(
     Ok((server_config, cert_chain_pem))
 }
 
-pub fn rt(name: String) -> Runtime {
+pub fn rt(name: String, num_threads: NonZeroUsize) -> Runtime {
```

Comment:
I think runtime should be created in at least validator.rs level and should be configured there correspondingly. This way we create all runtimes in one place, which allows:
* use one test runtime in unit-tests
* have a transparent configuration in one place
* propagate less values down the call stack

Reply:
Agreed with Lijun; let's keep this PR focused on wiring up control for the runtime and then shift around where it is instantiated later.

> I don't mind if this is split into two PRs, but in this case we will have to first introduce the number of threads in this config and later remove it.

I think that is fine, but l'm also inclined to think the majority of the plumbing can stay in place as-is. Ie, the plumbing from `agave-validator` CLI to the TPU seemingly might stay in place

Reply:
Streamer is using now PacketBytes which is just wrapper for Bytes (@vadorovsky chabge). So should be no memcpy.